packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -112,7 +123,18 @@ export const BitcoinOptions = () => {
                             </StyledButton>
                         </Tooltip>
                     )}
-                    <Tooltip content={<Translation id="BROADCAST_TOOLTIP" />} cursor="pointer">
+                    <Tooltip
+                        content={
+                            <Translation
+                                id={
+                                    broadcastDisabled
+                                        ? 'BROADCAST_TOOLTIP_DISABLED_LOCKTIME'
+                                        : 'BROADCAST_TOOLTIP'
+                                }
+                            />
+                        }
+                        cursor="pointer"
```

Comment:
Looks like this is ignored.

Reply:
Interesting. It is not ignored by the Tooltip component itself, but because Tooltip contains only `button` which has the `pointer` cursor it is always pointer. I will remove it from here.